fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -706,7 +706,7 @@ class CDUPerformancePage {
             }
         };
 
-        const closeToDest = mcdu.flightPlanManager.getDestination() && mcdu.flightPlanManager.getDestination().liveDistanceTo <= 180;
+        const closeToDest = mcdu.flightPlanManager.getDistanceToDestination() !== -1 && mcdu.flightPlanManager.getDistanceToDestination() <= 180;
```

Comment:
```suggestion
        const distanceToDestination = mcdu.flightPlanManager.getDistanceToDestination();
        const closeToDest = distanceToDestination >= 0 && distanceToDestination <= 180;
```
The call is relatively expensive, so maybe just call once.